lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
+                <% } else { %>
+                    <p>To access your sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
+                <% } %>
                 <p>
-                <a class="button-link" href="<%=signInUrl%>">Go to your DfE Sign-in account</a>
+                <a class="button-link" href="<%=signInUrl%>">Go to DfE Sign-in account</a>
```

Comment:
as per the prototype, this should be `Go to DfE Sign-in` and not `Go to DfE Sign-in account`

Reply:
fixed in next checkin